src/controller/scoringmgr/mock_internal/mock_lib_add.go

```diff
@@ -27,19 +27,8 @@ wrap_add(void *f, int a, int b){
 import "C"
```

Comment:
Please exclude this file. 
I think we should talk about a refactoring of `src/controller/scoringmgr/mock_internal` folder through a new issue or a new PR.

Reply:
@t25kim This is not a safe operation. 
https://golang.org/pkg/unsafe/#Pointer item(2) "Conversion of a uintptr back to Pointer is not valid in general. "

Reply:
I think most of files in [mock_internal](https://github.com/lf-edge/edge-home-orchestration-go/tree/9e1ebceefd2751c64e90815b76d367658a8d633e/src/controller/scoringmgr/mock_internal) folder are unnecessary and would like to remove them.
I know you point and let's discuss about refactoring of the folder after this PR.

Reply:
Do we need to create a separate ticket (issue) to handle the refactoring of the `src/controller/scoringmgr/mock_internal` folder after approving this as it is?

Reply:
> 
> 
> Do we need to create a separate ticket (issue) to handle the refactoring of the `src/controller/scoringmgr/mock_internal` folder after approving this as it is?

Yes, we can create a new issue. Yes, we can create a new issue. Once this PR is approved, we can add code validation to the Action-> workflow with `wet`

I think it's very important to add a static analyzer **SonarCloud** at the moment. This will help us take the next step to improve our code and also reach 100% of CII Best Practice.